Model/RefundFailedProcessor.php

```diff
@@ -65,7 +65,7 @@ public function processFailedWebhook($params, $jobName)
     {
         $data = $params[0];
         $json = !empty($data) ? json_decode($data, true) : [];
-        $refundId = isset($json['data']['refundId']) ? $json['data']['refundId'] : '';
```

Comment:
@dhiraj-toptal we don't have a test to update related to this?

Reply:
I have updated some Unit tests